src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,15 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public List<Lotto> getLottos() {
+        return lottos;
```

Comment:
저는 getter 사용시에는 외부에서 해당 객체의 변경을 일으킬 수 없도록 조심하는 편인데요.
이 경우 역시 `getLottos()`를 통해 가져온 `List`에 값을 추가하는 등의 변경이 가능할 것 같습니다. 
다음과 같이 불변 리스트를 반환해 보는 건 어떨까요?
```suggestion
        return Collections.unmodifiableList(lottos);
```

또는, 수정을 하더라도 해당 객체는 영향 받지 않도록 새로운 객체를 생성하여 반환해 줄 수도 있을 것 같아요.